mep/people/templates/people/member_detail.html

```diff
@@ -0,0 +1,51 @@
+{% extends 'base.html' %}
+
+{% block page-subtitle %}{{ member.name }} · {% endblock %}
+
+{% block header %}
+{% include 'snippets/header.html' with title=member.name %}
+{% endblock %}
+
+{% block content %}
+<nav class="tabs">
+    <li class="tab" aria-selected="true">
+        <a href="{% url 'people:member-detail' slug=member.mep_id %}">personography</a>
+    </li>
+    <li class="tab" aria-selected="false">
+        <a>books borrowed</a>
+    </li>
+    <li class="tab" aria-selected="false">
+        <a>lending cards</a>
+    </li>
+</nav>
+<section class="personography">
+    <dl>
+        {% if member.birth_year or member.death_year %}
+        <div class="dates">
+            <dt>Dates</dt>
+            <dd>{{ member.birth_year }} - {{ member.death_year }}</dd>
+        </div>
+        {% endif %}
+        {% if member.nationalities.count > 0 %}
+        <div class="nationalities">
+            <dt>Nationality</dt>
+            {% for nation in member.nationalities.all %}
+            <dd>{{ nation }}</dd>
+            {% endfor %}
+        </div>
+        {% endif %}
+        {% if member.viaf_id %}
+        <div class="reference">
+            <dt>Reference</dt>
+            <dd><a href="https://viaf.org/viaf/{{ member.viaf_id }}">VIAF</a></dd>
```

Comment:
viaf id is actually a URL already - this is duplicating the viaf.org portion of the url